CHANGELOG/v1.14.4.md

```diff
@@ -0,0 +1,19 @@
+## Changes by Kind
+
+### Bug or Regression
+
+- Don't allow fleetsMembers to have capital letters. Default fleet spec.group. ([#4818](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4818), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
+- [`AZURE_SYNC_PERIOD`](https://azure.github.io/azure-service-operator/guide/aso-controller-settings-options/#azure_sync_period) for ASO now defaults to `1h` instead of "effectively never" ([#4783](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4783), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
```

Comment:
```suggestion
- [`AZURE_SYNC_PERIOD`](https://azure.github.io/azure-service-operator/guide/aso-controller-settings-options/#azure_sync_period) for ASO now defaults to `1h` instead of "effectively never" ([#4783](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4783), [@nojnhuh](https://github.com/nojnhuh))
```